### PR TITLE
For #266: RtPlugin#enable and #disable tests.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #266: 
- Added tests for two conditions of `RtPlugin#enable` and `RtPlugin#disable`;
- Left todos for the rest of  implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#267](https://github.com/amihaiemil/docker-java-api/issues/267) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#267](https://github.com/amihaiemil/docker-java-api/issues/267) is assigned to @bkuzmic[/z](https://www.0crat.com/u/bkuzmic), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/519b1190-6377-4850-9508-1abd1bd28196); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/com/amihaiemil/docker/RtPluginTestCase.java [hunk 1]
```diff
@@ -106,4 +110,158 @@ public void inspectsItself() throws Exception {
...
+                    "Method should be a POST",
+                    req -> "POST".equals(req.getRequestLine().getMethod())
+                ),
+                new Condition(
+                    "Resource path must be /{name}/enable",
+                    req -> req.getRequestLine().getUri()
+                    .endsWith("/sshfs/enable")
+                )
+            ),
+            URI.create("http://localhost/plugins/sshfs/enable"),
```
Comment (bkuzmic) [inline src/test/java/com/amihaiemil/docker/RtPluginTestCase.java]: @paulodamaso URI of the RtPlugin should end with name. `/enable` is something that will be added by calling the `enable() `method. So please remove` /enable` from the end.

File: src/test/java/com/amihaiemil/docker/RtPluginTestCase.java [hunk 2]
```diff
@@ -106,4 +110,158 @@ public void inspectsItself() throws Exception {
...
```
Comment (bkuzmic) [inline src/test/java/com/amihaiemil/docker/RtPluginTestCase.java]: @paulodamaso Same as above. Remove `/enable` from the URI

File: src/test/java/com/amihaiemil/docker/RtPluginTestCase.java [hunk 3]
```diff
@@ -106,4 +110,158 @@ public void inspectsItself() throws Exception {
...
+                    "Method should be a POST",
+                    req -> "POST".equals(req.getRequestLine().getMethod())
+                ),
+                new Condition(
+                    "Resource path must be /{name}/disable",
+                    req -> req.getRequestLine().getUri()
+                    .endsWith("/sshfs/disable")
+                )
+            ),
+            URI.create("http://localhost/plugins/sshfs/disable"),
```
Comment (bkuzmic) [inline src/test/java/com/amihaiemil/docker/RtPluginTestCase.java]: @paulodamaso Same as above. Remove `/disable` from the end of URI.

File: src/test/java/com/amihaiemil/docker/RtPluginTestCase.java [hunk 4]
```diff
@@ -106,4 +110,158 @@ public void inspectsItself() throws Exception {
...
```
Comment (bkuzmic) [inline src/test/java/com/amihaiemil/docker/RtPluginTestCase.java]: @paulodamaso Same as above. Remove `/disable` from the end of URI.

**Comment by paulodamaso** [comment]
@bkuzmic Corrections done, please take a look

**Comment by bkuzmic** [comment]
@rultor merge it

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Job was finished in 47 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#267](https://github.com/amihaiemil/docker-java-api/issues/267) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @bkuzmic[/z](https://www.0crat.com/u/bkuzmic)

**Pull request merged**